content/id/docs/contribute/localization_id.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Dokumentasi Khusus Untuk Translasi Bahasa Indonesia
+content_type: concept
+---
+
+<!-- overview -->
+
+Panduan khusus untuk bergabung ke komunitas SIG DOC Indonesia dan melakukan 
```

Comment:
let's make it consistent with kubernetes community (https://kubernetes.io/docs/contribute/#get-involved-with-sig-docs)
```suggestion
Panduan khusus untuk bergabung ke komunitas SIG Docs Indonesia dan melakukan 
```

Reply:
Ok!